docs/workflows.md

```diff
@@ -101,63 +101,83 @@ Examples:
+For both options, a popup will open and the list of the available actions will show. You can search on the top for a specific action or browse the list by expanding the categories. After you choose an action by clicking on it, the configuration popup will show. Click on `Save` at the bottom of the popup to save the action after you configure it. Configuring the action is not required at this point and it can be saved blank.
 
-#### `How to configure them`
+#### `How to Configure Them`
 
-After you add an action you can click on it and a small icon set will appear on the right. The Action settings button is the wrench. By clicking it an settings popup will show up and you start configuring the action. When you are done click on save at the bottom of the popup to save the action.
+After you add an action you can click on it and a small icon set will appear on the right. The Action settings button is the wrench. By clicking it, the  settings popup will show and you may start configuring the action. When you are done, click on `Save` at the bottom of the popup to save the action.
 
-#### `How to access input parameters`
+#### `How to Access Input Parameters`
 
-Input parameters can be accessed as tokens directly in the actions (eg: [MyParam]).Keep in mind that the entities input parameters are expanded and you can access all the data of the entity (eg: [MyEntityParam:Property] or [ MyEntityParam :Id])
+Input parameters can be accessed as tokens directly in the actions (eg: [MyParam]). Keep in mind that the entities input parameters are expanded and you can access all the data of the entity (eg: `[MyEntityParam:Property]` or `[MyEntityParam:Id]`)
 
-#### `How to access other data through actions or tokens`
+#### `How to Access Other Data Through Actions or Tokens`
 
-Other data can be accessed with specific actions that read data from different sources. For example you can read other entities data based on a Server Request. This data came into the execution context without being based as an input parameter. Some actions that we can list in this category: Run SQL Query, Server Request, Read Entity Data also can come from Tokens (the module). By passing some of the input parameters to the token for example it can bring back a HTML code that can be used in Generate PDF action to obtain a PDF document.
+Other data can be accessed with specific actions that read data from different sources. For example, you can read other entities' data based on a Server Request. This data comes into the execution context without being specified as an input parameter. Some actions that we can list in this category include `Run SQL Query`, `Server Request`, and `Read Entity`. Data also can come from Plant an App Tokens. By passing some of the input parameters to the token, for example, it can bring back HTML code that can be used in a `Generate PDF` action to obtain a PDF document.
```

Comment:
```suggestion
Other data can be accessed with specific actions that read data from different sources. For example, you can read other entities' data based on a Server Request. This data comes into the execution context without being specified as an input parameter. Some actions in this category include `Run SQL Query`, `Server Request`, and `Read Entity`. Data also can come from Plant an App Tokens by passing some of the input parameters to the token. For example, it can bring back HTML code that can be used in a `Generate PDF` action to obtain a PDF document.
```